packages/plugin-board/test/integration/spec/board.js

```diff
@@ -0,0 +1,406 @@
+        });
+    });
+  });
+
+  describe(`#getContents()`, () => {
+
+    afterEach(() => participants[0].spark.board.deleteAllContent(board));
+
+    it(`adds and gets contents from the specified board`, () => {
+      const contents = [{type: `curve`}];
+      const data = [{
+        type: contents[0].type,
+        payload: JSON.stringify(contents[0])
+      }];
+
+      return participants[0].spark.board.addContent(conversation, board, data)
+        .then(() => {
+          return participants[0].spark.board.getAllContent(board);
+        })
+        .then((res) => {
+          assert.equal(res[0].payload, data[0].payload);
+        });
+    });
+
+    flaky(it)(`can deal with tons of contents by pagination`, () => {
+      const tonsOfContents = generateTonsOfContents(2100);
+
+      return participants[0].spark.board.addContent(conversation, board, tonsOfContents)
+        .then(() => {
+          return participants[0].spark.board.getAllContent(board);
+        })
+        .then((res) => {
+          assert.equal(res.length, tonsOfContents.length);
+          for (let i = 0; i < res.length; i++) {
+            assert.equal(res[i].payload, tonsOfContents[i].payload);
+          }
+        });
+    });
+  });
+
+  describe(`#deleteContent()`, () => {
+
+    after(() => participants[0].spark.board.deleteAllContent(board));
+
+    it(`delete contents from the specified board`, () => {
+      const channel = board;
+      const contents = [
+        {
+          id: uuid.v4(),
+          type: `file`
+        },
+        {
+          id: uuid.v4(),
+          type: `string`
+        }
+      ];
+      const data = [
+        {
+          type: contents[0].type,
+          payload: JSON.stringify(contents[0])
+        },
+        {
+          type: contents[1].type,
+          payload: JSON.stringify(contents[1])
+        }
+      ];
+
+      return participants[0].spark.board.addContent(conversation, channel, data)
+        .then(() => {
+          return participants[0].spark.board.deleteAllContent(channel);
+        })
+        .then(() => {
+          return participants[0].spark.board.getAllContent(channel);
+        })
+        .then((res) => {
+          assert.equal(res.length, 0);
+          return res;
+        })
+        .then(() => {
+          return participants[0].spark.board.addContent(conversation, channel, data);
+        })
+        .then((res) => {
+          assert.equal(res[0].items.length, 2);
+          const content = res[0].items[0];
+          console.log(`contentId: `, content.contentId);
+          return participants[0].spark.board.deleteContent(channel, content);
+        })
+        .then(() => {
+          return participants[0].spark.board.getAllContent(channel);
+        })
+        .then((res) => {
+          assert.equal(res.length, 1);
+          assert.equal(res[0].payload, data[1].payload);
+          return res;
+        });
+    });
+  });
+
+  describe(`realtime`, () => {
```

Comment:
I did this, but it leads to duplicated setup code.  That is why I consolidated in the first place.  Either way is great with me.